src/main/kotlin/no/nav/familie/ks/sak/kjerne/brev/begrunnelser/BegrunnelseType.kt

```diff
@@ -6,6 +6,7 @@ enum class BegrunnelseType(val sorteringsrekkefølge: Int) {
     INNVILGET(2),
     EØS_INNVILGET(2),
     AVSLAG(3),
+    EØS_AVSLAG(3),
```

Comment:
Tror denne må inn i Vedtaksperiode sin AVSLAG(listOf(BegrunnelseType.AVSLAG)),

Reply:
BegrunnelseType.AVSLAG brukes også flere steder i BrevPeriodeContext. Må man gjøre endringer der også for å støtte EØS_AVSLAG?

Reply:
Bra spotta! 🤩 

Jeg la inn EØS_AVSLAG der det skal trenges, de fleste steder AVSLAG blir brukt er NasjonalEllerFellesBegrunnelse spesifikk logikk